lib/middleware/csp.js

```diff
@@ -24,21 +45,49 @@ function createMiddleware(sCspUrlParameterName, oConfig) {
 		defaultPolicyIsReportOnly = false,
 		defaultPolicy2 = null,
 		defaultPolicy2IsReportOnly = false,
-		definedPolicies = {}
+		definedPolicies = {},
+		serveCSPReports = false
 	} = oConfig;
 
+
+	/**
+	 * List of CSP Report entries
+	 */
+	const cspReportEntries = [];
+
 	return function csp(req, res, next) {
 		const oParsedURL = parseurl(req);
 
 		if (req.method === "POST" ) {
 			if (req.headers["content-type"] === "application/csp-report" &&
-				oParsedURL.pathname.endsWith("/dummy.csplog") ) {
-				// In report-only mode there must be a report-uri defined
-				// For now just ignore the violation. It will be logged in the browser anyway.
+				oParsedURL.pathname.endsWith("/.ui5/csp/report.csplog") ) {
```

Comment:
As the report-uri is absolute, this check should compare the whole URL
```suggestion
				oParsedURL.pathname === "/.ui5/csp/report.csplog") {
```

Reply:
done